extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/deployment/LiquibaseProcessor.java

```diff
@@ -154,15 +154,14 @@ void nativeImageConfiguration(
                 new ReflectiveClassBuildItem(true, true, true, classesMarkedWithDatabaseChangeProperty.toArray(new String[0])));
 
         Collection<String> dataSourceNames = jdbcDataSourceBuildItems.stream()
-                .map(i -> i.getName())
+                .map(JdbcDataSourceBuildItem::getName)
                 .collect(Collectors.toSet());
 
         resource.produce(
                 new NativeImageResourceBuildItem(getChangeLogs(dataSourceNames, liquibaseBuildConfig).toArray(new String[0])));
 
         Stream.of(liquibase.change.Change.class,
                 liquibase.changelog.ChangeLogHistoryService.class,
-                liquibase.command.LiquibaseCommand.class,
```

Comment:
This is deprecated, hence also this change: https://github.com/quarkusio/quarkus/pull/26977/files#diff-930f0d68c059b4088f05d8c3d7a725e110279966c77958a6d699f9c652bde81eL48-R47

It was added back in #9636, but I don't see how the following is still the case today:
> This capability is also shown in our Liquibase quickstart